be/src/exec/schema_scanner.h

```diff
@@ -91,7 +91,7 @@ class SchemaScanner {
 public:
     struct ColumnDesc {
```

Comment:
I think we can just replace this with `SlotDescriptor` for simplicity.

Reply:
SlotDescriptor is an Object which is related to specified SQL execution, but ColumnDesc is related to some table. They are different things.

Reply:
SlotDescriptor is harder to contruct.

Reply:
@imay Isn't information schema a view instead of a table_